content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -506,15 +528,16 @@ The pipeline uses a number of prebuilt, reusable components, including:
 * The [tfma 
   component](https://github.com/kubeflow/pipelines/blob/master/components/dataflow/tfma/src/model_analysis.py)
   which writes out the data for the `web-app` viewer.
-* The [dataflow predict 
-  component](https://github.com/kubeflow/pipelines/blob/master/components/dataflow/predict/src/predict.py)
-  which writes out the data for the `table` viewer.
 
-## Usage in lightweight python components
+## Lightweight Python component Notebook example
 
-For lightweight components, the syntax is slightly different. You can refer to
+For a complete example of lightweigh Python component, cou can refer to
 [the lightweight python component notebook example](https://github.com/kubeflow/pipelines/blob/master/samples/core/lightweight_component/lightweight_component.ipynb) to learn more about declaring output visualizations.
 
+## yaml component example
+
+You can also configure visualization in a component.yaml file. See `{name: MLPipeline UI Metadata}` output in [component.yaml](https://github.com/kubeflow/pipelines/blob/f61048b5d2e1fb5a6a61782d570446b0ec940ff7/components/tensorflow/tensorboard/prepare_tensorboard/component.yaml#L12).
```

Comment:
nit: `See` is not an inclusive word, use `Refer to` instead.
Suggest changing `component.yaml` to `Create Tensorboard Visualization component` as an example, because it's best practice to describe the link's content using link text.

Reply:
Sounds good, thank you Yuan!